src/TrueLayer/ApiClient.cs

```diff
@@ -41,7 +41,11 @@ public ApiClient(HttpClient httpClient, IOptions<TrueLayerOptions> options)
         }
 
         /// <inheritdoc />
-        public async Task<ApiResponse<TData>> GetAsync<TData>(Uri uri, string? accessToken = null, CancellationToken cancellationToken = default)
+        public async Task<ApiResponse<TData>> GetAsync<TData>(
```

Comment:
OT: do we want to start enforcing formatting at CI? (I don't _think_ we do, IIRC)

Reply:
I was thinking of improving the `.editorconfig` file with extra configuration but I would just wait until everything else is done (refactoring including). In this specific case, it is just my own preference as I don't like seeing long lines and it also makes the review easier on GH given that people won't have to scroll to see the whole line.

Reply:
Yes, sure let's wait the next phase!